extra/mender-gateway/docker-compose.test.yml

```diff
@@ -0,0 +1,13 @@
+version: '2.1'
+services:
+  mender-gateway:
+    image: registry.mender.io/mendersoftware/mender-gateway:master
+    networks:
+      - mender
+      - mender_local
+    volumes:
+      - "./extra/mender-gateway/mender-gateway.conf:/etc/mender/mender-gateway.conf:ro"
```

Comment:
Note,

This volume (and the config file) to be removed after  [MEN-5300](https://tracker.mender.io/browse/MEN-5300). The base config file will be taken from `mender-gateway` repository examples.